extensions/container-image/container-image-docker/deployment/src/main/java/io/quarkus/container/image/docker/deployment/DockerProcessor.java

```diff
@@ -303,6 +303,7 @@ private String[] getDockerArgs(String image, DockerfilePaths dockerfilePaths, Co
             dockerArgs.add("--network");
             dockerArgs.add(network);
         });
+        dockerConfig.additionalArgs.ifPresent(dockerArgs::addAll);
```

Comment:
Does it still matter how `dockerArgs` is initialized? (`new ArrayList<>(6 + dockerConfig.buildArgs.size())`)?